### ISSUE TITLE
USER ISSUE: Game keep crashing

### ISSUE THREAD

**Issue by boundbythecurve** [StrangeLoopGames/EcoIssues#11925]
**Version:** 0.8.1.0 beta

**Steps to Reproduce:**
Start game. Watch it crash

**Expected behavior:**
Not crash

**Actual behavior:**
The following error message popped up:
```
Connection timed out
```

**Do you have mods installed? Does issue happen when no mods are installed?:**
The following error message popped up:
```
Connection timed out
```

**Comment by beastyn** [comment]
Did you try to create a new world or to connect to an existing server?

**Comment by boundbythecurve** [comment]
Sorry I wasn't sure why the game kept crashing when I made the report. I
would start the world and within 1 minute it would crash. It was a somewhat
new world, but not brand new.
But I think I figured it out. Each time I started the game, I was reset to
a time where I was adding a resource to a project in a campfire. However,
even though I have permission to use this campfire, the item being cooked
was not started by me. It was started by the other player in my game: my
wife. So I think that was the problem. The two times the game crashed, I
had put resource (I think it was wood pulp) to a project already in
progress, but short on resources. And that project was not started by me.
The third time I restarted the world, I didn't add the resources. I just
dropped them on the ground and let my wife add them, and everything was
fine. The game has been not crashing ever since.
